newton/_src/solvers/vbd/rigid_vbd_kernels.py

```
@@ -1103,11 +1103,24 @@ def evaluate_cable_stretch_shear_force_hessian(
     force = f_world if is_parent else -f_world
 
     t = _quat_rotate_local_z(q_wp)
-    K_eff = h_s * wp.identity(3, float) + (h_z - h_s) * wp.outer(t, t)
-    rx = wp.skew(r)
+    I = wp.identity(3, float)
```

Ruff (0.16.0)

[error] Ambiguous variable name: `I`

(E741)